common/djangoapps/util/password_policy_validators.py

```diff
@@ -156,43 +264,51 @@ def validate_password_complexity(value):
 
     errors = []
     if len(uppercase) < complexities.get("UPPER", 0):
-        errors.append(_("must contain {0} or more uppercase characters").format(complexities["UPPER"]))
+        errors.append('UPPER')
     if len(lowercase) < complexities.get("LOWER", 0):
-        errors.append(_("must contain {0} or more lowercase characters").format(complexities["LOWER"]))
+        errors.append('LOWER')
     if len(digits) < complexities.get("DIGITS", 0):
-        errors.append(_("must contain {0} or more digits").format(complexities["DIGITS"]))
+        errors.append('DIGITS')
     if len(punctuation) < complexities.get("PUNCTUATION", 0):
-        errors.append(_("must contain {0} or more punctuation characters").format(complexities["PUNCTUATION"]))
+        errors.append('PUNCTUATION')
     if len(non_ascii) < complexities.get("NON ASCII", 0):
-        errors.append(_("must contain {0} or more non ascii characters").format(complexities["NON ASCII"]))
+        errors.append('NON ASCII')
     if len(words) < complexities.get("WORDS", 0):
-        errors.append(_("must contain {0} or more unique words").format(complexities["WORDS"]))
+        errors.append('WORDS')
     if len(numeric) < complexities.get("NUMERIC", 0):
-        errors.append(_("must contain {0} or more numbers").format(complexities["NUMERIC"]))
+        errors.append('NUMERIC')
     if len(alphabetic) < complexities.get("ALPHABETIC", 0):
-        errors.append(_("must contain {0} or more letters").format(complexities["ALPHABETIC"]))
+        errors.append('ALPHABETIC')
 
     if errors:
-        raise ValidationError(message.format(u', '.join(errors)), code=code)
+        return _password_complexity_descriptions(errors)
+    else:
+        return []
 
 
-def validate_password_against_username(password, username):
+def _validate_password_against_username(password, username):
+    if not username:
+        return
+
     if password == username:
         # Translators: This message is shown to users who enter a password matching
         # the username they enter(ed).
-        raise ValidationError(_(u"Password cannot be the same as the username"))
+        raise ValidationError(_(u"Password cannot be the same as the username."))
 
 
-def validate_password_dictionary(value):
+def _validate_password_dictionary(value):
```

Comment:
Just double checking that these methods weren't used externally and don't need to be updated (though I believe you looked)

Reply:
Yes, I believe I checked for all of them. Although if I missed any, tests will let me know. (Tests are failing now because I forgot to update messages across all the tests. Working on that.)